src/TwilioCallMessage.php

```diff
@@ -91,4 +96,17 @@ public function fallbackMethod(string $fallbackMethod): self
 
         return $this;
     }
+
+    /**
+     * Set the answering machine detection mode.
+     *
+     * @param string $machineDetection
+     * @return $this
+     */
+    public function machineDetection(string $machineDetection): self
```

Comment:
Maybe worth adding an enum as well for the two supported values here?
https://www.twilio.com/docs/voice/answering-machine-detection#teconsent